app/views/logs/new.html.erb

```diff
@@ -24,6 +24,15 @@
               </select>
             </div>
 
```

Comment:
Whats the commented out code for? We cant leave it in if we pushing it to master